### ISSUE TITLE
Template part toolbar contains an input that's not keyboard accessible

### ISSUE THREAD

**Issue by talldan** [WordPress/gutenberg#28825]
## Description
The following e2e test is failing (https://github.com/WordPress/gutenberg/runs/1852467249):
```
FAIL specs/experiments/document-settings.test.js (12.378 s)

Document Settings › when a template is selected from the navigation sidebar › and a template part is clicked in the template › should display the selected template part's name in the document header

    expect(jest.fn()).not.toHaveWarned(expected)

    Expected mock function not to be called but it was called with:
    ["Using custom components as toolbar controls is deprecated. Please use ToolbarItem or ToolbarButton components instead. See: https://developer.wordpress.org/block-editor/components/toolbar-button/#inside-blockcontrols"]
```

The test seems to have caught an actual bug, something in a toolbar hasn't been implemented using `ToolbarItem`, which is required for the toolbar's keyboard accessibility. The screenshot shows this, so it seems related to the template part toolbar:
![document-settings-when-a-template-is-selected-from-the-navigation-sidebar-and-a-template-part-is-clicked-in-the-template-should-display-the-selected-template-parts-name-in-the-document-header-2021-02-08t06-46-53](https://user-images.githubusercontent.com/677833/107186752-cebdae80-6a1f-11eb-817b-81e6adc076ca.jpg)

**Comment by talldan** [comment]
Closing as the toolbar for template part no longer has an input (https://github.com/WordPress/gutenberg/pull/27397#issuecomment-784069932)
